modules/claims_api/app/sidekiq/claims_api/report_hourly_unsuccessful_submissions.rb

```diff
@@ -8,15 +8,15 @@ class ReportHourlyUnsuccessfulSubmissions < ClaimsApi::ServiceBase
     def perform
       return unless allow_processing?
 
-      @search_to = 1.minute.ago
+      @search_to = Time.zone.now
```

Comment:
I remember a ticket that made some attempt of not falsely alerting as we thought there may have been some issue with submissions coming in at the same time as the hourly report, and somehow falsely reporting errored status to our Slack alert - is the change from 1.minute.ago to now a needed fix? 

we've also modified it [here](https://github.com/department-of-veterans-affairs/vets-api/pull/18606/files) - perhaps this is needed for this query to work, just want to make sure we're not re-introducing something we intentionally changed?